server/sonar-server/src/test/java/org/sonar/server/user/ws/SearchActionTest.java

```diff
@@ -192,16 +227,12 @@ public void search_with_groups() throws Exception {
         .setName(name)
         .setScmAccounts(scmAccounts)
         .setLocal(true)
-        .setUpdatedAt(createdAt)));
+        .setExternalIdentity(login)
+        .setExternalIdentityProvider("sonarqube")
+        .setUpdatedAt(createdAt));
+      userDtos.add(userDto);
 
-      users[index] = new UserDoc()
-        .setActive(true)
-        .setCreatedAt(createdAt)
-        .setEmail(email)
-        .setLogin(login)
-        .setName(name)
-        .setScmAccounts(scmAccounts)
-        .setUpdatedAt(createdAt);
+      users[index] = toUserDoc(userDto);
```

Comment:
To prevent having to create UserDoc, you know that you can use the UserIndexer (This feedback is not to be applied now).